ephios/extra/mixins.py

```diff
@@ -119,3 +121,18 @@ def get_canonical_slug(self):
         object class. In that case, this method will never be called.
         """
         return self.get_object().slug
+
+
+class PermissionFormMixin:
```

Comment:
Can this be in the same file as the corresponding field and get a docstring that they work together please?

Reply:
it has been that way, i thought you'd prefer having all mixins in mixins.py ^^